### ISSUE TITLE
dependency updates

### ISSUE THREAD

**Issue by juliasloan25** [CliMA/ClimaCoupler.jl#554]
update to ClimaAtmos v0.18, ClimaLSM 0.5

closed by #556

### notes
- merge after #538 is fixed
- ClimaLSM v0.5.0 is incompatible with NCDatasets v0.14, so we'll have to update NCDatasets in another PR after LSM releases next
